code/grant/pokedex/pokedex/management/commands/load_pokemon.py

```diff
@@ -0,0 +1,1953 @@
+#         },
+#         {
+#             "number": 147,
+#             "name": "dratini",
+#             "height": 18,
+#             "weight": 33,
+#             "image_front": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/147.png",
+#             "image_back": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/back/147.png",
+#             "types": [
+#                 "dragon"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/dratini"
+#         },
+#         {
+#             "number": 148,
+#             "name": "dragonair",
+#             "height": 40,
+#             "weight": 165,
+#             "image_front": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/148.png",
+#             "image_back": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/back/148.png",
+#             "types": [
+#                 "dragon"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/dragonair"
+#         },
+#         {
+#             "number": 149,
+#             "name": "dragonite",
+#             "height": 22,
+#             "weight": 2100,
+#             "image_front": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/149.png",
+#             "image_back": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/back/149.png",
+#             "types": [
+#                 "flying",
+#                 "dragon"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/dragonite"
+#         },
+#         {
+#             "number": 150,
+#             "name": "mewtwo",
+#             "height": 20,
+#             "weight": 1220,
+#             "image_front": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/150.png",
+#             "image_back": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/back/150.png",
+#             "types": [
+#                 "psychic"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/mewtwo"
+#         },
+#         {
+#             "number": 151,
+#             "name": "mew",
+#             "height": 4,
+#             "weight": 40,
+#             "image_front": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/151.png",
+#             "image_back": "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/back/151.png",
+#             "types": [
+#                 "psychic"
+#             ],
+#             "url": "https://pokemon.fandom.com/wiki/mew"
+#         }
+#     ]
+# }
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+
+        Pokemon.objects.all().delete()
+        PokemonType.objects.all().delete()
+
+        f = open('pokemon.json')
+
+        # with open('pokemon.json', 'w') as poke_balls:
+
+        #     json.dump(f, poke_balls)
+
+        print(f)
+
+        poke_balls = json.load(f)
```

Comment:
Not a huge issue, but a context manager could be used to avoid having to call `close()` at the end. 
```python
with open('pokemon.json', 'r') as pokemon_json_file:
    json_string = pokemon_json_file.read()
    # do more stuff with the file open
```